main.js

```diff
@@ -0,0 +1,53 @@
+import Imperial from "./imperial.js";
+import log from "./schnelleinsteigLog.js";
+
+Vue.component("player", {
+  props: ["name", "cash", "bonds"],
+  template: `
+  <li class="player">
+    <div class="contents">
+      <h3>{{ name }}</h3>
+      <div>Cash: {{ cash }} million</div>
+      <div>
+        Bonds:
+        <ul class="bonds">
+          <bond
+            v-for="bond in bonds"
+            v-bind:nation="bond.nation.value"
+            v-bind:cost="bond.cost"
+            v-bind:key="bond.nation"
+          ></bond>
+        </ul>
+      </div>
+    </div>
+  </li>
+  `,
+});
+
+Vue.component("bond", {
+  props: ["nation", "cost"],
+  template: `<li class="bond">{{ nation }}{{ cost }}</li>`,
+});
```

Comment:
I'm unfamiliar with Vue. How would we have written this in React+JSX? Like this?

```es6
const bond = ({ nation, cost }) => <li class="bond">{nation}{cost}</li>
```

Reply:
I'm not sure. It's been so long since I've done React and I never did do it in the newer function-based syntax.